File: src/main/java/ru/siaw/free/rpplugin/function/Me.java
package ru.siaw.free.rpplugin.function;

import org.bukkit.entity.Player;
import ru.siaw.free.rpplugin.utility.RpSender;

public class Me {
    private static final RpSender sendUtil = new RpSender();

    public static void send(Player sender, String msg) {
        sendUtil.use(sender, msg);
    }

    public static void setMeOriginal(String value) {
        sendUtil.setOriginal(value);
    }

    public static void setMeGlobal(boolean value) {
        sendUtil.setGlobal(value);
    }

    public static void setMeRadius(int value) {
        sendUtil.setRadius(value);
    }
}

File: src/main/java/ru/siaw/free/rpplugin/function/Me.kt
package ru.siaw.free.rpplugin.function

import ru.siaw.free.rpplugin.utility.RpSender
import org.bukkit.entity.Player

class Me {
    private val msgSender = RpSender()
    fun send(sender: Player?, msg: String?) {
        msgSender.use(sender!!, msg!!)
    }

    fun setMeEnabled(value: Boolean) {
        msgSender.setMeEnabled(value)
    }

    fun setMeGlobal(value: Boolean) {
        msgSender.setMeGlobal(value)
    }

    fun setMeRadius(value: Int) {
        msgSender.setMeRadius(value.toDouble())
    }

    fun setMeMsg(value: String?) {
        msgSender.setMeMsg(value!!)
    }
}